cmake/modules/BuildSPDK.cmake

```diff
@@ -10,7 +10,7 @@ macro(build_spdk)
     find_package(uuid REQUIRED)
   endif()
 
-  find_program (MAKE_EXECUTABLE NAMES make gmake)
+  find_program (MAKE_EXECUTABLE NAMES gmake make)
```

Comment:
could you take this chance to

1. remove the space after `find_program`
2. error out if `MAKE_EXECUTABLE` evaluates to "false", please note if "gmake" or "make" is not found, "${MAKE_EXECUTABLE}" will be "MAKE_EXECUTABLE-NOTFOUND". and cmake consider it as a "false".